app/src/main/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivity.kt

```diff
@@ -125,4 +123,11 @@ class QuestionPlayerActivity :
   override fun dismissConceptCard() {
     questionPlayerActivityPresenter.dismissConceptCard()
   }
+
+  override fun deleteCurrentProgressAndStopSession() {
+    questionPlayerActivityPresenter.stopTrainingSession()
+  }
+
+  // This function is not needed because the no progress is saved in a training session.
+  override fun deleteOldestProgressAndStopSession() {}
```

Comment:
These are the two methods that have names that can be a bit misleading in QuestionPlayerActivity.